.github/workflows/license-checker.yml

```diff
@@ -3,10 +3,10 @@ name: License checker
 on:
   push:
     branches:
-      - master
+      - feature/distribute-reorg
   pull_request:
     branches:
-      - master
+      - feature/distribute-reorg
```

Comment:
I think this should be reverted? Otherwise LGTM

Reply:
It will merge to feature branch. So we should modify it to enable license checker.